config/default.yml

```diff
@@ -558,6 +558,16 @@ RSpec/LetSetup:
   VersionAdded: '1.7'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/LetSetup
 
+RSpec/MemoizedHelperBlockDelimiter:
+  Description: Use consistent style block delimiters for memoized helpers.
+  Enabled: pending
+  EnforcedStyle: do_end
```

Comment:
I disagree with the choice of the default style.

`do_end` doesn't seem to be a frequent choice in the wild, according to a run against `real-world-rspec`:
```
57497 files inspected, 104170 offenses detected, 104170 offenses autocorrectable
```
as opposed to `braces`:
```
57497 files inspected, 17121 offenses detected
```

And, I suspect some part of do/end usages (offences to `braces` style) are due to the line length limitation:
```ruby
    let!(:conference) do
      WimbaConference.create!(title: "my conference", user: @user, duration: 60, context: course_factory)
    end
```

or because of being multi-line e.g. `/Users/pirj/source/real-world-rspec/spree/emails/spec/mailers/spree/order_mailer_spec.rb:25:22:`:
```ruby
  let(:second_order) do
    order = create(:completed_order_with_totals, email: 'test2@example.com')
    product = create(:product, name: %{The "BESTEST" product})
    variant = create(:variant, product: product)
    price = create(:price, variant: variant, amount: 15.00)
    store = second_store
    line_item = create(:line_item, variant: variant, order: order, quantity: 1, price: 4.99)
    allow(product).to receive_messages(default_variant: variant)
    allow(variant).to receive_messages(default_price: price)
    allow(order).to receive_messages(line_items: [line_item])
    allow(order).to receive_messages(store: store)
    order
  end
```